<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoWidget</class>
 <widget class="QWidget" name="VideoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>12</y>
     <width>711</width>
     <height>524</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioButton_all">
        <property name="maximumSize">
         <size>
          <width>116</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_mine">
        <property name="maximumSize">
         <size>
          <width>116</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>mine</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_other">
        <property name="maximumSize">
         <size>
          <width>116</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>other</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="minimumSize">
         <size>
          <width>501</width>
          <height>0</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>num</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>title</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>size</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>time</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>file</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="2">
           <widget class="QLabel" name="labelName">
            <property name="maximumSize">
             <size>
              <width>104</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string extracomment="app name label">Title</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" rowspan="3">
           <widget class="QLabel" name="labelIcon">
            <property name="minimumSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>72</width>
              <height>72</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="pixmap">
             <pixmap resource="../zasoby.qrc">:/icons/info.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="labelQRcode">
            <property name="maximumSize">
             <size>
              <width>9</width>
              <height>72</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="editTitle">
            <property name="maximumSize">
             <size>
              <width>104</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="labelVersion">
            <property name="maximumSize">
             <size>
              <width>104</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string extracomment="version label">File Name</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="editName">
            <property name="maximumSize">
             <size>
              <width>104</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QLabel" name="labelAppsFileName">
            <property name="maximumSize">
             <size>
              <width>199</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string extracomment="file name label">File Path</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="3">
           <widget class="QLineEdit" name="editFilePath">
            <property name="maximumSize">
             <size>
              <width>199</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="0" colspan="3">
           <widget class="QLabel" name="labelAppsSize">
            <property name="maximumSize">
             <size>
              <width>199</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string extracomment="size label">Size</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="3">
           <widget class="QLineEdit" name="editSize">
            <property name="maximumSize">
             <size>
              <width>199</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="3">
           <widget class="QLabel" name="labelAppsLocation">
            <property name="maximumSize">
             <size>
              <width>199</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string extracomment="location label">Duration</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="3">
           <widget class="QLineEdit" name="editDuration">
            <property name="maximumSize">
             <size>
              <width>199</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_in">
          <property name="maximumSize">
           <size>
            <width>199</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>import</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_out">
          <property name="maximumSize">
           <size>
            <width>199</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>export</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete">
          <property name="maximumSize">
           <size>
            <width>199</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_refresh">
          <property name="maximumSize">
           <size>
            <width>199</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="maximumSize">
           <size>
            <width>199</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="format">
           <string>%p%</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="msg_label">
          <property name="maximumSize">
           <size>
            <width>199</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
